### PR TITLE
adjust toniebox cards to style of tonie cards and make small ux changed

### PR DESCRIPTION
**Pull request by CTeipen**
image

image

image

image

image

image

### CODE REVIEW

**Comment by SciLor** [comment]
This looks nice, but there is some trouble with the intendation and the quotes.
Please be sure you check the commits changes before commiting them. This will flood the diffs and make following the changes very difficult.

File: public/translations/de.json [hunk 1]
```diff
@@ -1,214 +1,220 @@
...
+    "tonieboxes": {
+        "navigationTitle": "Tonieboxen",
+        "title": "Deine Tonieboxen",
+        "lastPlayedTonie": "Zuletzt gespielter Tonie: ",
+        "editModelModal": {
+            "editModel": "Toniebox \"{{name}}\" bearbeiten",
+            "placeholderSearchForAModel": "Wähle deine Farbvariante aus",
+            "name": "Bezeichnung",
+            "model": "Modell",
+            "unsetBoxName": "Unbekannte Box",
```
Comment (henryk86) [inline public/translations/de.json:154]: Modell unbekannt

File: public/translations/en.json [hunk 2]
```diff
@@ -1,213 +1,219 @@
...
+    "tonieboxes": {
+        "navigationTitle": "Tonieboxes",
+        "title": "Your Tonieboxes",
+        "lastPlayedTonie": "Last played Tonie: ",
+        "editModelModal": {
+            "editModel": "Edit Toniebox \"{{name}}\"",
+            "placeholderSearchForAModel": "Select color variant of your Toniebox",
+            "name": "Name",
+            "model": "Model",
+            "unsetBoxName": "Unknown Box",
```
Comment (henryk86) [inline public/translations/en.json:153]: Model unknown/not set

**Review by henryk86 — APPROVED** [review]

File: public/translations/en.json [hunk 3]
```diff
@@ -139,7 +139,7 @@
       "selectFile": "Select TAF-file"
     },
     "informationModal": {
-      "unknownModel": "Unknown Model",
+      "unknownModel": "Unknwown Model",
```
Comment (SciLor) [inline public/translations/en.json:142]: Reintroduced typo

File: public/translations/de.json [hunk 4]
```diff
@@ -44,6 +44,7 @@
   "tonies": {
     "navigationTitle": "Tonies",
     "title": "Tonies",
+    "unsetTonie": "Undefinierter Tonie",
```
Comment (henryk86) [inline public/translations/de.json:47]: Sounds kind of negative. Would only state „Tonie“. Same in English

Comment (SciLor) [reply public/translations/de.json:47]: I think it would be enough just naming it Tonie or Tonie/Tag.

**Pull request merged**